src/drivers/vmount/output_mavlink.cpp

```diff
@@ -54,7 +54,7 @@ OutputMavlink::OutputMavlink(const OutputConfig &output_config)
 
 int OutputMavlink::update(const ControlData *control_data)
 {
-	vehicle_command_s vehicle_command;
+	vehicle_command_s vehicle_command = {};
```

Comment:
This is forcing an initialisation on every iteration - is this really required? Benchmarking PX4 we found that the system is spending quite a bit of time in these defensive programming patterns. Which are generally good, but induce a lot of load (5-8% CPU IIRC) in embedded.